package-overrides/npm/angular-aria@1.5.0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "format": "global",
+  "jspmNodeConversion": false,
+  "main": "angular-aria",
+  "dependencies": {
+    "angular": "npm:angular@^1.5.0"
```

Comment:
As this PR won't influence the default github (i.e. bower) based ecosystem for Angular I thought that this change shouldn't cause compatibility issues. By introducing the Angular npm namespace we've just to make sure that each overwrite sticks explicitly to the npm ecosystem regarding Angular as otherwise you'll end up with a bower and npm installation of Angular in your application. But so far my published overwrites are respecting that.

If this changes are too critical for you I can add them also in my private jspm registry but I just thought that it might help others who want to use the npm version of Angular as well.